### PR TITLE
feat(docs): update examples to stravalib 2.x

### PR DESCRIPTION
**Pull request by lwasser**
This is a pr to update the activities documentation page so it aligns with the new version of stravalib. 

## Type of change

Select the statement best describes this pull request.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] This is a documentation update
- [ ] This is a infrastructure update (docs, ci, etc)
- [ ] Other (please describe)

## Does your PR include tests

If you are fixing a bug or adding a feature, we appreciate (but do not require)
tests to support whatever fix of feature you're implementing.

- [ ] Yes
- [x] No, i'd like some help with tests
- [ ] This change doesn't require tests

## Did you include your contribution to the change log?

- [x] Yes, `changelog.md` is up-to-date.

### CODE REVIEW

File: docs/get-started/activities.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+(activities)=
```
Comment (lwasser) [inline docs/get-started/activities.md:1]: This isn't a full rewrite. i moved this file from rst to myst md. then i edited it!

File: docs/_static/stravalib.css [hunk 2]
```diff
@@ -2,6 +2,11 @@ html, body {
     font-size: 1.02rem;
 }
 
+/* We are using comments for code output so make them more visible */
+html[data-theme="dark"] .highlight .go {
```
Comment (lwasser) [inline docs/_static/stravalib.css:6]: This doesn't work - in dark mode comments are really light.i was trying to brighten them a but but i may have to experiment a bit more.

**Review by jsamoocha — APPROVED** [review]
Thanks again! This looks good to merge. Reviewing was challenging because of the lack of a diff. If there are any discrepancies, we'll learn in time :-)

File: docs/_static/stravalib.css [hunk 3]
```diff
@@ -2,6 +2,11 @@ html, body {
...
+    color: #cbc673!important;
+}
```
Comment (lwasser) [inline docs/_static/stravalib.css:8]:
```suggestion

```

File: docs/conf.py [hunk 4]
```diff
@@ -112,6 +112,9 @@
     # "navbar_align": "left",  # [left, content, right] For testing that the navbar items align properly
     "github_url": "https://github.com/stravalib/stravalib",
     "footer_start": ["copyright"],
+    # https://pygments.org/styles/
+    "pygments_light_style": "xcode",
+    "pygments_dark_style": "github-dark",
```
Comment (lwasser) [inline docs/conf.py:117]:
```
```

File: docs/conf.py [hunk 5]
```diff
@@ -112,6 +112,9 @@
...
     "announcement": "<a href='whats-new/stravalib-2.html'>Stravalib 2.x is out 🚀! Check out our migration guide for tips on changes from Stravalib V1!</a>",
```
Comment (lwasser) [inline docs/conf.py:118]:
```suggestion
    "announcement": "<a href='whats-new/stravalib-2.html'>Stravalib 2.x is out 🚀! Check out our migration guide for tips on changes from Stravalib V1!</a>",
```

File: docs/_static/stravalib.css [hunk 6]
```diff
@@ -2,6 +2,8 @@ html, body {
     font-size: 1.02rem;
 }
 
+/* We are using comments for code output so make them more visible */
+
 .admonition {
```
Comment (lwasser) [inline docs/_static/stravalib.css:7]:
```suggestion

.admonition {
```

**Comment by lwasser** [comment]
this is ready for review. I will play with the code highlighting settings in a separate PR. I think that some of the colors are not accessible to users. we are using default code highlight settings. I may have to either customize or select a different highlighting theme. I just didn't find one that looked accessible - specifically the comments tend to be a darker color than i'd prefer!

**Comment by lwasser** [comment]
> Thanks again! This looks good to merge. Reviewing was challenging because of the lack of a diff. If there are any discrepancies, we'll learn in time :-)

 i didn't consider that! i noticed it looked like a new file but you are right. I'll merge and am happy to make changes as needed in the future. I did a bit of reading. I could try next time to rename it in a separate commit and then add another commit that is ONLY the text changes to see if then we'd get a useful diff.  🤷🏻‍♀️ 
 
Merging now. thank you for the review!

**Pull request merged**